libs/utils/utils.js

```diff
@@ -693,6 +693,7 @@ export function decorateLinks(el) {
   const { hostname, href } = window.location;
   const links = [...anchors].reduce((rdx, a) => {
     appendHtmlToLink(a);
+    if (a.href.includes('http:')) a.setAttribute('data-http-link', 'true');
```

Comment:
Any particular reason to use the data attribute here? We can just do the detection in `seo.js`.